index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Rock Paper Scissors</title>
+</head>
+<header>
+    <h1>Rock, Paper, Scissors</h1>
+</header>
+<body>
+  <main>
+    <p id="intro">An easily memorised rule determines the winner: Rock breaks scissors, scissors cuts paper, paper covers rock. In other words, the one who chooses rock beats the one who chooses scissors; scissors in turn beats paper; and paper beats rock. Both players choosing the same results in a tie</p>
+
+    <label>
+      <input type="radio" name="player-choice" value="rock" checked>
+      <img src="https://images.freeimages.com/images/small-previews/cef/on-da-rocks-1177316.jpg"  alt="rock">
+    </label>
+    <br></br>
```

Comment:
Looks like you're using br tags to create a spacing between your labels. You should be using CSS to style your app.

Reply:
I will do that on the next one...how though?